src/main/java/io/kestra/plugin/nats/ProduceInterface.java

```diff
@@ -0,0 +1,29 @@
+package io.kestra.plugin.nats;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+import java.util.Map;
+
+public interface ProduceInterface {
```

Comment:
This is not needed as only the Produce task needs those properties so you can define them inside the task directly

Reply:
Me not, browsing an interface just to read the annotations is cumbersome.